transformers4rec/torch/model/base.py

```diff
@@ -543,7 +553,17 @@ def forward(self, inputs: TensorOrTabularData, training=True, **kwargs):
             )
 
         if len(outputs) == 1:
-            return outputs[list(outputs.keys())[0]]
+            outputs = outputs[list(outputs.keys())[0]]
+            if isinstance(outputs, dict):
+                # next-item prediction task with `hf_format = True``  returns a dictionary
+                # with three tensors: loss, predictions, labels. This needed for training
+                # and evaluation using the Trainer class.
+                # At inference, we just need the predictions tensors.
+                # TODO: We are simplifying the logic around `hf_format` in the multi-gpu
+                # support work.
+                if not training and not self.hf_format:
```

Comment:
Can we remove `not training` from here so that the `hf_format` controls the output. That would make it work in systems without a model adapter wrapper class